docs/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:8.11.4
+
+WORKDIR /app/website
+
+EXPOSE 3000 35729
+COPY ./docs /app/docs
+COPY ./website /app/website
+RUN yarn install
+
+CMD ["yarn", "start"]
```

Comment:
I think you can use a standard node image and mount docs and website as volumes in the main docker-compose of the project.